src/frontend/src/lib/components/ui/TextWithDescriptionAndLogo.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
```

Comment:
I'm not a fund of component's code duplication I have to say. Wouldn't it be cleaner to extend `TextWithLogo` either by:

- Adding the property `description` directly within that component

or

- Adding a `slot` in `TextWithLogo` and then using this component in `TextWithDescriptionAndLogo`

WDYT?

Reply:
Yes, in the end re-using `TextWithLogo` doesn't end up in a lot of `if`s as I'm initially was afraid of, just a few adjustments and now this component can accept `description` as a prop. Thanks for the idea 👍